### DISCUSSION TITLE
Ideas, Questions, Anything

### DISCUSSION THREAD

**Discussion by mechatroner** [mechatroner/rainbow_csv#24] [General]
Hello World!

**Comment by nashidau** [comment]
Can I use rainbow_csv to convert a TSV to a CSV file?

**Comment by mechatroner** [comment]
@nashidau 
Yes, you can do it with RBQL:
1. Temporarily set RBQL output format as CSV `:let g:rbql_output_format='csv'`
2. Run `:SELECT *`

**Comment by PatProMath** [comment]
I have a csv file with no delimiter but with a fixed width/length for each row. Rainbow CSV is not recognizing the file though. 

@mechatroner Is there a feature to do this? With allowance to decide the width of each row and remove unnecessary trailing spaces? Or even convert from the fixed length CSV to a separator-delimited CSV file.
Or is it something you could work on for another version?
